src/pages/moderator/paddle/adjustments.tsx

```diff
@@ -0,0 +1,210 @@
+            value={filters.transactionId?.join(',') ?? ''}
+            onChange={(e) =>
+              setFilters({
+                ...filters,
+                transactionId: e.target.value ? e.target.value.split(',') : undefined,
+              })
+            }
+            size="sm"
+            disabled={isLoading}
+          />
+        </Group>
+        <Select
+          label="Type"
+          name="size"
+          data={[
+            { value: 'all', label: 'All' },
+            ...AdjustmentAction.map((action) => ({
+              value: action as string,
+              label: toPascalCase(action.replace('_', ' ')),
+            })),
+          ]}
+          value={filters.action ?? 'all'}
+          onChange={(value: string) =>
+            setFilters({
+              ...filters,
+              action: value === 'all' ? undefined : (value as (typeof AdjustmentAction)[number]),
+            })
+          }
+          disabled={isLoading}
+        />
+      </Group>
+
+      {isLoading ? (
+        <Center p="xl">
+          <Loader size="lg" />
+        </Center>
+      ) : !!adjustments.length ? (
+        <Stack>
+          <Table>
+            <thead>
+              <tr>
+                <th>Type</th>
+                <th>Created at</th>
+                <th>Currency</th>
+                <th>Amount</th>
+                <th>Status</th>
+                <th>Customer</th>
+                <th>Reason</th>
+              </tr>
+            </thead>
+            <tbody>
+              {adjustments.map((adjustment) => {
+                return (
+                  <tr key={adjustment.id}>
+                    <td>{toPascalCase(adjustment.action.replace('_', ' '))}</td>
+                    <td>{formatDate(adjustment.createdAt)}</td>
+                    <td>{adjustment.currencyCode ?? 'N/A'}</td>
+                    <td>
+                      {adjustment.payoutTotals?.total
+                        ? formatPriceForDisplay(parseInt(adjustment.payoutTotals?.total, 10))
+                        : 'N/A'}
+                    </td>
+                    <td>
+                      <Badge>{getDisplayName(adjustment.status.replace('_', ' '))}</Badge>
+                    </td>
+                    <td>
+                      <Stack spacing={0}>
+                        <Anchor
+                          size="xs"
+                          // Not keen on this approach, but will have to do in the meantime.
+                          href={`https://vendors.paddle.com/customers-v2/${adjustment.customerId}`}
+                          target="_blank"
+                        >
+                          Paddle
+                        </Anchor>
+                        <Anchor
+                          size="xs"
+                          href={`/moderator/paddle/customer/${adjustment.customerId}`}
+                          target="_blank"
+                        >
+                          Civitai
+                        </Anchor>
```

Comment:
Let's wrap this in a `<Link>` component
```suggestion
                       <Link href={`/moderator/paddle/customer/${adjustment.customerId}`} passHref legacyBehavior>
                            <Anchor
                              size="xs"
                            target="_blank"
                          >
                            Civitai
                          </Anchor>
                        </Link>
```